extensions/wikia/Wall/Wall.i18n.php

```diff
@@ -4231,14 +4287,29 @@
 	'wall-topic-heading' => 'このスレッドのタグ:',
 	'wall-topic-edit' => 'タグを編集',
 	'wall-topic-edit-save' => '保存',
-	'wall-topic-edit-cancel' => '中止',
-	'group-threadmoderator' => 'モデレーター',
-	'group-threadmoderator-member' => 'モデレーター',
-	'grouppage-threadmoderator' => 'w:c:community:ヘルプ:モデレーター',
-	'wall-message-edited-summary' => '<a href="$4" >$1 by </a><a class="username" href="$2">$3</a>',
-];
+	'wall-topic-edit-cancel' => 'キャンセル',
+	'group-threadmoderator' => 'ディスカッション・モデレーター',
+	'group-threadmoderator-member' => 'ディスカッション・モデレーター',
+	'grouppage-threadmoderator' => '',
+	'wall-message-edited-summary' => '<a href="$4" >$1</a>（編集：<a class="username" href="$2">$3さん</a>）',
+	'wall-message-limit-reached' => 'スレッドあたりの投稿数の上限は$1件までです。',
+	'wall-action-monobook-login' => 'Please log in to post a message on this wiki',
+	'wall-posting-message-failed-title' => 'メッセージの投稿中にエラーが発生しました',
+	'wall-posting-message-failed-body' => 'メッセージをコピーして、後ほどもう一度お試しください',
+	'tog-enotifmywall-reminder' => '毎日のリマインダーメールも受け取る',
+	'right-walledit' => 'Can edit wall posts',
+	'right-wallarchive' => 'Can archive wall posts',
+	'right-wallremove' => 'Can remove wall threads',
+	'right-wallshowwikiaemblem' => 'Flags user as "Wikia Staff" on wall posts',
```

Comment:
Good catch! I see that the old version also has Wikia used, so I guess we need to adjust the translations in crowdin and repull them. I'll ping Keiko.